src/api/config/environments/development.rb

```diff
@@ -8,8 +8,8 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Do not eager load code on boot.
-  config.eager_load = false
+  # Eager load code on boot.
+  config.eager_load = true
```

Comment:
Thanks for the review @ChrisBr! Just to check, are you saying you don't mind having `config.eager_load = true` or you don't mind using `::Project` in the webui controllers?

Also FYI, there is a third option too:

3. Change the folder name to `app/controllers/webui/projects/` so the module doesn't conflict with the model. This is the preferred way for organising rails controllers but it wouldn't be consisted with the existing controller and route naming like `ProjectController`